include/arbor/util/compat.hpp

```diff
@@ -37,4 +37,14 @@ inline void compiler_barrier_if_icc_leq(unsigned ver) {
 #endif
 }
 
+// Work-around for bad vectorization of fma in gcc version < 8.2
+
+template <typename T>
+#if defined(__GNUC__) && (100*__GNUC__ + __GNUC_MINOR__ < 802)
```

Comment:
Have you tested for gcc 8.1? I suppose gcc 8.x isn't available on daint, so I assume that you verified 8.2 by looking at the assembly generated for your reduced reproducer on your desktop?
It would be nice to draw a line at gcc 8.

Reply:
The 8.2 test was based on the comment on the gcc bug report, but correct behaviour in the arbor unit test has not been verified with 8.2 — I didn't want to have to install gcc on greina :)